src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -477,7 +480,11 @@ export const Checkout = ({
         locale: locale.isoLocale,
         quoteCartId,
         quotes: getUniqueQuotesFromVariantList(quoteBundleVariants).map(
-          ({ startDate, currentInsurer, data: { type, typeOfContract } }) => {
+          ({
+            startDate,
+            currentInsurer,
+            data: { type, typeOfContract, isStudent },
+          }) => {
```

Comment:
`isStudent` isn't returned for SE apartment/house quotes tho -- so is this really working?

My solution looked like this:

```
  const mainQuote = bundleSelector.getMainQuote(selectedQuoteBundle.bundle)
  ...

  const initialValues = {
    ...
    data: {
      ...mainQuoteData,
      isStudent: quoteSelector.isStudent(mainQuote),
    },
  } as QuoteInput
```

Reply:
I think `isStudent` is actually returned with the new quote cart API

<img width="525" alt="Screenshot 2022-03-02 at 10 21 40" src="https://user-images.githubusercontent.com/42962286/156332797-7087eaac-4bcd-4a8e-8d81-9b988cab2a45.png">

Reply:
Or maybe that's not what you meant @robinandeer 🙈

Reply:
hmm, I didn't think it was.. but if it is, that makes things a lot easier!